lib/collection/src/shards/local_shard.rs

```diff
@@ -58,6 +58,23 @@ pub struct LocalShard {
 
 /// Shard holds information about segments and WAL.
 impl LocalShard {
+    pub async fn move_data(from: &Path, to: &Path) -> CollectionResult<()> {
+        let wal_from = Self::wal_path(from);
+        let wal_to = Self::wal_path(to);
+        let segments_from = Self::segments_path(from);
+        let segments_to = Self::segments_path(to);
+        tokio::fs::rename(wal_from, wal_to).await?;
+        tokio::fs::rename(segments_from, segments_to).await?;
```

Comment:
Should we rollback the wal renaming if the segment renaming fails?

Reply:
what if rollback fails?